game-core/src/main/java/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -105,25 +105,22 @@ public static File getUserRootFolder() {
    *     installations. Users can override this location in settings.
    */
   public static File getUserMapsFolder() {
-    final File mapsFolder =
-        getUserMapsFolder(ClientSetting.userMapsFolderPath, ClientSetting.mapFolderOverride)
-            .toFile();
-    if (!mapsFolder.exists()) {
-      mapsFolder.mkdirs();
-    }
-    if (!mapsFolder.exists()) {
-      log.severe("Error, downloaded maps folder does not exist: " + mapsFolder.getAbsolutePath());
-    }
-    return mapsFolder;
+    return getUserMapsFolder(ClientFileSystemHelper::getRootFolder);
```

Comment:
`getRootFolder()` is a little bit weird tbh.
On a clean system the triplea folder will never be inside "documents", only if you manually create this folder it will end up storing it there indefinitely.